### ISSUE TITLE
Scrollable Frame issue 

### ISSUE THREAD

**Issue by UserJoo9** [TomSchimansky/CustomTkinter#1193]
tried to use Scrollable frame in another frame and tabview and its keep give label and fg_color exception i don't know why?

**Comment by TomSchimansky** [comment]
Can you please share an example where the error occurs, for me everything is working.

**Comment by iaZe** [comment]
using this:
``` 
scrollable_frame = customtkinter.CTkScrollableFrame(master, width=200, height=200)
scrollable_frame.grid(row=0, column=0, padx=10, pady=10)
```
I had this return:
 ```
ctk_scrollable_frame.py", line 64, in __init__ fg_color=ThemeManager.theme["CTkScrollableFrame"]["label_fg_color"] if label_fg_color is None else label_fg_color)
KeyError: 'CTkScrollableFrame'
```

**Comment by iaZe** [comment]
I performed some tests and noticed that if you set the theme to dark-blue the frame doesn't work anymore, you can only set the appearance e por isso retorna esse erro
```
customtkinter.set_default_color_theme("dark-blue")
```

**Comment by TomSchimansky** [comment]
Should be fixed now with 5.1.2.
